cmd/ratify/cmd/referrer.go

```diff
@@ -37,7 +36,7 @@ const (
 type referrerCmdOptions struct {
 	configFilePath string
 	subject        string
-	artifactTypes  []string
+	artifactType   string
```

Comment:
Hi @akashsinghal , i m not familiar with the referrer command, what is the previous sample usage for a  array of artifact type. Can you clarify how does it work with a single artifact type? thanks!

Reply:
So actually I don't think it was used previously. I searched the referrer.go file and it's only declared but never used. I believe it's left over from the discover/verify commands that use that flag too.